lib/js/src/compose/types/page-block/geometry/feed.ts

```diff
@@ -53,4 +53,8 @@ export default class Feed {
       }
     }
   }
+
+  isValid (): boolean {
+    return this.options.moduleID !== NoID
```

Comment:
This is not the only thing needed in the feed for it to be valid. Try configuring it and see, might also be worth adding a * next to the required values (in the form next to labels))

Otherwise this is the right approach